### PR TITLE
eslint fix demo (do not merge)

### PR DESCRIPTION
**Pull request by settinghead**

### CODE REVIEW

**Comment by settinghead** [comment]
@alex-deaconu @ezequielc This PR is for demonstration purpose: it's the result of auto-fixing some of the linting issues via eslint (very similar to beautify). Thanks.

File: src-js/Viewer/Utils.js [hunk 1]
```diff
@@ -53,19 +51,19 @@ utils.showElement = function(elementName, show) {
...
+utils.reportViewerConfigToPlayer = function () {
   var viewerVersion = global.VIEWER_VERSION,
-  width = window.innerWidth,
-  height = window.innerHeight; 
+    width = window.innerWidth,
+    height = window.innerHeight;
 
   try {
-    window.postToPlayer({ message: 'viewer-config', viewerVersion: viewerVersion, width: width, height: height });
+    window.postToPlayer({ message: 'viewer-config', viewerVersion, width, height });
```
Comment (alex-deaconu) [inline src-js/Viewer/Utils.js]: @settinghead was this automatically corrected? If so this functionality is now broken... Please take a look. Thanks.

Comment (settinghead) [inline src-js/Viewer/Utils.js]: @alex-deaconu actually it's an ES6 shorthand, and is correct.

Comment (alex-deaconu) [inline src-js/Viewer/Utils.js]: Got it, thanks for the clarification!

File: src-js/index.js [hunk 2]
```diff
@@ -1,23 +1,23 @@
 var RiseVision = {
-	Viewer: {
-	  EntryPoint: require('./Viewer/EntryPoint'),
-		Channel: {
-			ChannelConnectionController: require('./Viewer/Channel/ChannelConnectionController'),
+	                                        Viewer: {
```
Comment (settinghead) [inline src-js/index.js]: @alex-deaconu yeah this one looks odd. Will look into it.

Comment (settinghead) [inline src-js/index.js]: @alex-deaconu This was probably due to a weird bug on space/indentation processing. However I replaced tabs with spaces in this file and linting results became correct.

File: src-js/Viewer/EntryPoint.js [hunk 3]
```diff
@@ -40,256 +40,254 @@ var dataReadyCommand;
...
-  
-  type = utils.getFromQueryString(TYPE_PARAM);
-  if (type == null) {
-    type = service.DISPLAY;
+  // example: queryString = '?type=display&id=1d3b23d3-31f6-4ad8-b8b1-c180231b9919';
+  // example: queryString = '?type=display&id=1d3b23d3-31f6-4ad8-b8b1-c180231b9919&CompanyAuthKey=';
+  // example: queryString = '?type=preview'; **** DEPRECATED
+  // example: queryString = '?type=presentation&id=1d3b23d3-31f6-4ad8-b8b1-c180231b9919&showui=false';
+
+  																				type = utils.getFromQueryString(TYPE_PARAM);
```
Comment (alex-deaconu) [inline src-js/Viewer/EntryPoint.js]: @settinghead the extra tabs/spaces are back here and https://github.com/Rise-Vision/viewer/pull/117/files#diff-253d098814a5f55d19a2854cc723486aR2. Thanks.

Comment (settinghead) [inline src-js/Viewer/EntryPoint.js]: @alex-deaconu same issue, fixed now.

**Comment by settinghead** [comment]
@alex-deaconu @ezequielc btw FYI I disabled coveralls bot from commenting on PR's in this repo, because we are receiving too many notifications and we can also check the coverage percentage by just clicking on "Show All checks".

File: src-js/Viewer/Channel/ChannelConnectionController.js [hunk 4]
```diff
@@ -1,15 +0,0 @@
-var service = {};
```
Comment (alex-deaconu) [inline src-js/Viewer/Channel/ChannelConnectionController.js]: @settinghead to confirm, these files were deleted by mistake, not by the Eslint fix task? Thanks.

Comment (settinghead) [inline src-js/Viewer/Channel/ChannelConnectionController.js]: @alex-deaconu I redid the whole linting and committed, and doesn't seem to be any lost file now. So yeah I'd assume it was a mistake.

**Pull request closed without merging**